TipCatDotNet.Api/Controllers/PaymentController.cs

```diff
@@ -0,0 +1,49 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Web.Resource;
+using TipCatDotNet.Api.Models.HospitalityFacilities;
+using TipCatDotNet.Api.Services.HospitalityFacilities;
+
+namespace TipCatDotNet.Api.Controllers
+{
+    [AllowAnonymous]
+    [Route("api/payments")]
+    [Produces("application/json")]
+    [RequiredScope(ScopeRequiredByApi)]
+    public class PaymentController : BaseController
+    {
+        public PaymentController(IPaymentService paymentService)
+        {
+            _paymentService = paymentService;
+        }
+
+
+        /// <summary>
+        /// Gets receiver info by member code.
+        /// </summary>
+        /// <param name="memberCode">Member Code</param>
+        /// <returns></returns>
+        [HttpGet("{memberCode}")]
```

Comment:
As for the method name, I suggest to add  a meaning to the route, like `api/payments/{memberCode}/prepare`